tests/test_data.py

```diff
@@ -0,0 +1,71 @@
+from unittest import mock
+import pytest
+
+from app import data
```

Comment:
The data module no longer exists.

Reply:
Already obsolet. Accidentally push commits to early. :-)